docs/sphinx_pyodide/sphinx_pyodide/packages.py

```diff
@@ -51,12 +51,12 @@ def run(self):
         def parse_package_info(
             self, config: pathlib.Path
         ) -> tuple[str, str, bool, str]:
-            yaml_data = parse_package_config(config)
+            yaml_data = MetaConfig.from_yaml(config)
 
-            name = yaml_data["package"]["name"]
-            version = yaml_data["package"]["version"]
-            tag = yaml_data["package"].get("_tag", "")
```

Comment:
For some reason, pydantic errors on `_` prefixed fields, so aliased them to the names without prefixes. The meta.yaml is unchanged.